### PR TITLE
Adds inputs to account resource for `auto_enable_controls` and `enable_default_standards`

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
Results of `make test`:

```
AWS_DEFAULT_REGION=us-east-1 TF_VAR_member_email=...... make test PYTEST_ARGS="-v --nomock"
[bootstrap]: Using external tardigrade-ci, skipping auto-init
[pytest//home/loren/git/repos/tardigrade-ci/tests/terraform_pytest] Starting Python tests found under the directory "/home/loren/git/repos/tardigrade-ci/tests/terraform_pytest"
pytest /home/loren/git/repos/tardigrade-ci/tests/terraform_pytest -v --nomock
============================================ test session starts =============================================
platform linux -- Python 3.8.10, pytest-7.2.2, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
rootdir: /home/loren/git/repos
plugins: mock-3.6.1, cov-2.12.1, travis-fold-1.3.0
collected 2 items

../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[securityhub_cross_account] PASSED                                                                                                                                                            [ 50%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[securityhub_same_account] PASSED                                                                                                                                                             [100%]

======================================= 2 passed in 123.43s (0:02:03) ========================================
[pytest//home/loren/git/repos/tardigrade-ci/tests/terraform_pytest]: Tests executed!
```

File: modules/cross-account-member/variables.tf [hunk 1]
```diff
@@ -13,12 +13,24 @@ variable "action_targets" {
...
 }
 
+variable "auto_enable_controls" {
+  description = "Boolean that enables the security standards that Security Hub has designated as automatically enabled including: `AWS Foundational Security Best Practices v1.0.0` and `CIS AWS Foundations Benchmark v1.2.0`"
+  type        = bool
+  default     = true
+}
+
 variable "control_finding_generator" {
   description = "(Optional) Updates whether the calling account has consolidated control findings turned on."
```
Comment (eemperor) [inline modules/cross-account-member/variables.tf:23]: Update the description here to match the others?

**Review by eemperor — COMMENTED** [review]
Looks good.  I just have the one comment about the one description not matching the others.

File: modules/cross-account-member/variables.tf (hunk 1 above)
Comment (lorengordon) [reply modules/cross-account-member/variables.tf]: done

**Review by eemperor — APPROVED** [review]

**Pull request merged**